pipeline/transform.py

```diff
@@ -46,6 +43,7 @@ def get_earthquake_property(data:dict, property_name: str) -> int | str | None:
         logging.error(f'An unexpected error occurred: {e}')
         return None
```

Comment:
In this function you've used 'properties' a lot, I think it'll be better to declare a constant variable and use that instead to avoid magic strings `PROPERTIES = 'properties'